snatched-crawler-client/src/app/scrape/page.tsx

```diff
@@ -24,11 +24,14 @@ export default function ScraperForm() {
 
     
     try {
+      const token = localStorage.getItem('auth_token')
+      console.log("Token", token)
       // You can replace this with your actual API route
       const response = await fetch('http://localhost:5000/api/scrape', {
```

Comment:
separate base_url from the api path,use a constant for base_url since that can change